src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -40,50 +40,93 @@ public class DrivetrainSubsystem extends SubsystemBase {
   // right side encoder
   private RelativeEncoder m_encoder_Right = right1.getEncoder();
 
+  private double angles;
+  private double degrees;
+
   public DrivetrainSubsystem() {
     m_robotDrive.setSafetyEnabled(false);
     leftSide.setInverted(true);
   }
 
-  public void setbrake() {
+  public void setBrake() {
     right1.setIdleMode(IdleMode.kBrake);
     right2.setIdleMode(IdleMode.kBrake);
     left1.setIdleMode(IdleMode.kBrake);
     left2.setIdleMode(IdleMode.kBrake);
   }
 
-  public void setCoast() {}
+  public void setCoast() {
+    right1.setIdleMode(IdleMode.kCoast);
+    right2.setIdleMode(IdleMode.kCoast);
+    left1.setIdleMode(IdleMode.kCoast);
+    left2.setIdleMode(IdleMode.kCoast);
+  }
 
   public void zeroencoders() {
     m_encoder_Right.setPosition(0);
     m_encoder_Left.setPosition(0);
   }
 
-  public void moveforward() {
-    if (m_encoder_Right.getPosition() / 9.52 >= -1.5f) {
-      m_robotDrive.arcadeDrive(0.6, 0);
+  public void zeroyawnavx() {
+    navx.calibrate();
+  }
+
+  public boolean move(double inches) {
+    inches = inches / 18.8495559215;
```

Comment:
Can you leave a comment here about how this value was calculated in case it needs to be changed?